components/bl-credit-card-component/component.json

```diff
@@ -0,0 +1,245 @@
+{
+  "id": "c_96cfb7158384c3e5564554f4a24d511a",
+  "name": "Credit Card",
+  "description": "This is a payment form component with beautiful interface of credit cards.",
+  "showInToolbox": true,
+  "faIcon": "credit-card",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "checkbox",
+      "name": "cardPreviewVisibility",
+      "label": "Card Preview Visibility",
+      "showInSettings": true,
+      "hasLogicHandler": false,
+      "handlerId": "cardPreviewVisibilityLogic",
+      "defaultValue": true
+    },
+    {
+      "type": "checkbox",
+      "name": "cvcVisibility",
```

Comment:
does it hide the CVC input?

is it possible to use a card with the CVV code?

Reply:
It doesn't hide the CVC input. It only changes its type to `password` and replace each value with `*` in the preview.